azdev/operations/extensions/__init__.py

```diff
@@ -190,7 +190,7 @@ def update_extension_index(extensions):
     from .util import get_ext_metadata, get_whl_from_url
 
     ext_repos = get_ext_repo_paths()
-    index_path = next((x for x in find_files(ext_repos, 'index.json') if 'azure-cli-extensions' in x), None)
+    index_path = next((x for x in find_files(ext_repos, 'index.json')), None)
```

Comment:
I think this is to distinguish the main `azure-cli-extensions` repo and service team's own extension repo. Please check if there are other places that assume the main repo directory is named `azure-cli-extensions`. And we may need to make sure other extension repos like [batch](https://github.com/Azure/azure-batch-cli-extensions)do not have this `index.json`, although I think they are not likely to have one.